MediaBrowser.XbmcMetadata/Parsers/BaseNfoParser.cs

```diff
@@ -460,10 +460,28 @@ protected virtual void FetchDataFromXmlNode(XmlReader reader, MetadataResult<T>
                     var trailer = reader.ReadNormalizedString();
                     if (!string.IsNullOrEmpty(trailer))
                     {
-                        item.AddTrailerUrl(trailer.Replace(
-                            "plugin://plugin.video.youtube/?action=play_video&videoid=",
-                            BaseNfoSaver.YouTubeWatchUrl,
-                            StringComparison.OrdinalIgnoreCase));
+                        if (trailer.StartsWith("plugin://plugin.video.youtube/?action=play_video&videoid=", StringComparison.OrdinalIgnoreCase))
+                        {
+                            // Deprecated format
+                            item.AddTrailerUrl(trailer.Replace(
+                                "plugin://plugin.video.youtube/?action=play_video&videoid=",
+                                BaseNfoSaver.YouTubeWatchUrl,
+                                StringComparison.OrdinalIgnoreCase));
+
+                            var suggested_url = trailer.Replace(
```

Comment:
```suggestion
                            var suggestedUrl = trailer.Replace(
```

We don't use snake_case.